### DISCUSSION TITLE
How to build separate PIXI modules?

### DISCUSSION THREAD

**Discussion by ProgrammingLife** [pixijs/pixijs#8291] [General]
Trying to use separate PIXI modules like this:
```
import { Application } from '@pixi/app';
// ... import { Loader } .... etc
```

I try to compile it to js file so I can use it in a web browser:
```
$ git clone https://github.com/pixijs/pixijs.git 
$ cd pixijs/packages/app/
$ npm i 
$ npm install npm-run-all --save-dev
$ npm run build
```

After that I get a single js-file build (~1.6mb).
How to get separate PIXI modules in js format to include them on the webpage separately?

**Comment by bigtimebuddy** [comment]
There are two questions here.

1. How do I get a browser-based bundle of individual packages. We ship this build with npm, so it's accessible from a CDN and we also publish it on our own CDN pixijs.download. For example:
  * https://unpkg.com/@pixi/app@6.3.0/dist/browser/app.min.js
  * https://pixijs.download/v6.3.0/packages/app.min.js
  
2. How do I build an individual package? We don't support this workflow in our repo. You need to run `npm run watch` or `npm run build` to generate.

**Comment by miltoncandelero** [comment]
You don't need to build pixi by yourself if you want to use the modular import approach. Check https://pixijs.io/customize/ to pick your own packages and make sure you have a "glue" file that registers all plugins into the renderer,
